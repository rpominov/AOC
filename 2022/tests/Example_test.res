open Jest

test("example", () => {
  expect(Example.sum(1, 2))->toMatchSnapshot
})
