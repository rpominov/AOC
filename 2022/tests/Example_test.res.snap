// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example 1`] = `3`;
